setup.py

```diff
@@ -18,7 +18,10 @@
       author='Matheus Cardoso',
       author_email='matheus.mcas@gmail.com',
       license='MIT',
+      install_requires=[
+          'six==1.10.0'
```

Comment:
Have in mind that pinning the version here will need manual work or pyup.io to help maintaining the requirement updated.